contracts/SmartLoan.sol

```diff
@@ -170,12 +170,16 @@ contract SmartLoan is OwnableUpgradeable, PriceAwareUpgradeable {
   }
 
 
-  function getSolvencyRatio() public view returns(uint256) {
+  /**
+    * LoanToValue ratio is calculated as the ratio between debt and collateral.
+    * The collateral is equal to total loan value takeaway debt.
+  **/
+  function getLoanToValueRatio() public view returns(uint256) {
     uint256 debt = getDebt();
     if (debt == 0) {
-      return MAX_SOLVENCY_RATIO;
+      return MIN_LTV;
     } else {
-      return getTotalValue() * PERCENTAGE_PRECISION / debt;
+      return debt * PERCENTAGE_PRECISION / (getTotalValue() - debt);
```

Comment:
We need to be careful here:
1) getTotalValue() == debt (division by 0)
2) getTotalValue() < debt (underflow)

Let's check that precondition before calculation and return MAX_LTV in that case